lib/rex/ui/text/dispatcher_shell.rb

```diff
@@ -5,661 +5,650 @@
+            return "" if commands.nil? or commands.length == 0
+
+            # Display the commands
+            tbl = Rex::Text::Table.new(
+                'Header'  => "#{self.name} Commands",
+                'Indent'  => opts['Indent'] || 4,
+                'Columns' =>
+                    [
+                        'Command',
+                        'Description'
+                    ],
+                'ColProps' =>
+                    {
+                        'Command' =>
+                            {
+                                'MaxWidth' => 12
+                            }
+                    })
+
+            commands.sort.each { |c|
+              tbl << c
+            }
+
+            return "\n" + tbl.to_s + "\n"
+          end
 
-    #
-    # No tab completion items by default
-    #
-    attr_accessor :shell, :tab_complete_items
-
-    #
-    # Provide a generic tab completion for file names.
-    #
-    # If the only completion is a directory, this descends into that directory
-    # and continues completions with filenames contained within.
-    #
-    def tab_complete_filenames(str, words)
-      matches = ::Readline::FILENAME_COMPLETION_PROC.call(str)
-      if matches and matches.length == 1 and File.directory?(matches[0])
-        dir = matches[0]
-        dir += File::SEPARATOR if dir[-1,1] != File::SEPARATOR
-        matches = ::Readline::FILENAME_COMPLETION_PROC.call(dir)
-      end
-      matches
-    end
+          #
+          # Return the subdir of the `documentation/` directory that should be used
+          # to find usage documentation
+          #
+          # TODO: get this value from somewhere that doesn't invert a bunch of
+          # dependencies
+          #
+          def docs_dir
+            File.expand_path(File.join(__FILE__, '..', '..', '..', '..', '..', 'documentation', 'cli'))
+          end
 
-    #
-    # Return a list of possible directory for tab completion.
-    #
-    def tab_complete_directory(str, words)
-      str = '.' + ::File::SEPARATOR if str.empty?
-      dirs = Dir.glob(str.concat('*'), File::FNM_CASEFOLD).select { |x| File.directory?(x) }
+          #
+          # No tab completion items by default
+          #
+          attr_accessor :shell, :tab_complete_items
+
+          #
+          # Provide a generic tab completion for file names.
+          #
+          # If the only completion is a directory, this descends into that directory
+          # and continues completions with filenames contained within.
+          #
+          def tab_complete_filenames(str, words)
+            matches = ::Readline::FILENAME_COMPLETION_PROC.call(str)
+            if matches and matches.length == 1 and File.directory?(matches[0])
+              dir = matches[0]
+              dir += File::SEPARATOR if dir[-1,1] != File::SEPARATOR
+              matches = ::Readline::FILENAME_COMPLETION_PROC.call(dir)
+            end
+            matches
+          end
 
```

Comment:
I think there are no functional changes in this file? all the whitespace got changed